.github/workflows/pr-build.yml

```diff
@@ -21,23 +21,24 @@ jobs:
           - 12.x
           - 14.x
           - 15.x
+
     steps:
       - name: Checkout AWS XRay SDK Node Repository @ default branch latest
         uses: actions/checkout@v2
+
       - name: Setup Node ${{ matrix.node-version }}
         uses: actions/setup-node@v1
         with:
           node-version: ${{ matrix.node-version }}
+
       - name: Cache NPM modules
         uses: actions/cache@v2
         with:
-          path: |
-            node_modules
-            */*/node_modules
-          key: ${{ matrix.os }}-${{ hashFiles('**/package.json') }}
+          path: $HOME/.npm
```

Comment:
This seems natural to me for sure, but any idea why github actions would recommend node_modules?

https://github.com/actions/cache/blob/main/examples.md#node---lerna